src/models/viewer.ts

```diff
@@ -0,0 +1,9 @@
+export interface IUser {
```

Comment:
oops i didn't mean to commit this file. I'll pull it out before this PR gets merged.

Reply:
👍